### PR TITLE
Add new field on invoices/estimates/invoice_templates/estimate_templates/invoice_templates_gallery/estimate_templates_gallery

### PR DESCRIPTION
**Pull request by brudascacosmin**

### CODE REVIEW

File: sections/invoice_templates.md [hunk 1]
```diff
@@ -82,6 +86,7 @@ Example of response:
         {
             "id": 5,
             "name": "Deutsch - Client on left",
+            "title": "INVOICE",
```
Comment (paymo-old) [inline sections/invoice_templates.md:89]: RECHNUNG

File: sections/invoice_templates.md [hunk 2]
```diff
@@ -91,6 +96,7 @@ Example of response:
         {
             "id": 6,
             "name": "Deutsch - Client on right",
+            "title": "INVOICE",
```
Comment (paymo-old) [inline sections/invoice_templates.md:99]: in german

File: sections/invoice_templates.md [hunk 3]
```diff
@@ -100,6 +106,7 @@ Example of response:
         {
             "id": 13,
             "name": "Español - Client on left",
+            "title": "INVOICE",
```
Comment (paymo-old) [inline sections/invoice_templates.md:109]: in spanish

File: sections/invoice_templates.md [hunk 4]
```diff
@@ -109,6 +116,7 @@ Example of response:
         {
             "id": 14,
             "name": "Español - Client on right",
+            "title": "INVOICE",
```
Comment (paymo-old) [inline sections/invoice_templates.md:119]: in spanish

File: sections/invoice_templates.md [hunk 5]
```diff
@@ -232,6 +241,7 @@ Attribute|Type|Description
 ---------|----|-----------
 id | integer | _(read-only)_ Unique template identifier
 name | text | Template name
+title | text | Default invoice title (this is copied from invoice_templates_gallery when company add new invoice_template)
```
Comment (paymoapp) [inline sections/invoice_templates.md]: rephrase

**Pull request merged**